### PR TITLE
Update system image instructions

### PR DESCRIPTION
**Pull request by jkozdon**
- Adds scripts for building the system images for the julia formatter
- Adds examples for how to call the system image script for `ClimateMachine`

### CODE REVIEW

File: docs/src/DevDocs/CodeStyle.md [hunk 1]
```diff
@@ -86,36 +86,39 @@ from the `PackageCompiler` repository should be noted:
...
+
 ```
-$ julia -q
-julia> using Pkg
-julia> Pkg.add("PackageCompiler")
-julia> using PackageCompiler
-julia> Pkg.activate(joinpath(@__DIR__, ".dev"))
-julia> using PackageCompiler
-julia> PackageCompiler.create_sysimage(:JuliaFormatter; precompile_execution_file=joinpath(@__DIR__, ".dev/precompile.jl"), replace_default=true)
+julia .dev/clima_formatter_default_imate.jl
```
Comment (kpamnany) [inline docs/src/DevDocs/CodeStyle.md:94]:
```suggestion
julia .dev/clima_formatter_default_image.jl
```

Comment (jkozdon) [reply docs/src/DevDocs/CodeStyle.md]: Thanks

File: .dev/clima_formatter_default_image.jl [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    or use the script
+        $(abspath(joinpath(@__DIR__, "..", ".dev", "clima_formatter_image.jl")))
+    which will use a custom path for the system image
+    """
+    exit(1)
+end
+
+# If a current Manifest exist for the formatter we remove it so that we have the
+# latest version
+rm(joinpath(@__DIR__, "Manifest.toml"), force = true)
```
Comment (charleskawczynski) [inline .dev/clima_formatter_default_image.jl:27]:
```suggestion
rm(joinpath(@__DIR__, "Manifest.toml"); force = true)
```
It works, but I think it's going to be deprecated soon.

File: .dev/clima_formatter_image.jl [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    $(sysimage_path)
+"""
+
+using Pkg
+Pkg.add("PackageCompiler")
+using PackageCompiler
+
+# If a current Manifest exist for the formatter we remove it so that we have the
+# latest version
+rm(joinpath(@__DIR__, "Manifest.toml"), force = true)
```
Comment (charleskawczynski) [inline .dev/clima_formatter_image.jl:26]:
```suggestion
rm(joinpath(@__DIR__, "Manifest.toml"); force = true)
```

**Review by charleskawczynski — APPROVED** [review]
LGTM, thanks!

File: .dev/clima_formatter_default_image.jl (hunk 2 above)
Comment (jkozdon) [reply .dev/clima_formatter_default_image.jl]: Thanks

**Pull request merged**